packages/core/src/index.js

```diff
@@ -1,3 +1,12 @@
+import { loadNuxtConfig } from '@nuxt/config'
+import Nuxt from './nuxt'
```

Comment:
We have `export { default as Nuxt } from './nuxt'` below, now it can be :

```js
export Nuxt
```

Reply:
I was thinking this may ensure less breaking changes and the fact that we don't export something else by accident. Also fewer characters :)